### PR TITLE
WIP: MOI

### PR DESCRIPTION
**Pull request by joaquimg**
@odow any comments?

### CODE REVIEW

File: src/MOI/attributes.jl [hunk 1]
```diff
@@ -0,0 +1,259 @@
...
+    end
+    return true
+end
+function MOI.getattribute(m::XpressSolverInstance, primal::MOI.PrimalStatus)
+    if !is_mip(m.inner)
+        return MOI.FeasiblePoint
+    else
+        return MOI.FeasiblePoint
+    end
+end
```
Comment (odow) [inline src/MOI/attributes.jl]: We need to port some feasibility/unboundedness tests

File: src/MOI/attributes.jl [hunk 2]
```diff
@@ -0,0 +1,259 @@
...
+    end
+    return true
+end
+function MOI.getattribute(m::XpressSolverInstance, primal::MOI.DualStatus)
+    if !is_mip(m.inner)
+        return MOI.FeasiblePoint
+    else
+        return MOI.FeasiblePoint
+    end
+end
```
Comment (odow) [inline src/MOI/attributes.jl]: ditto. I realised that it's not actually that easy to map every CPLEX status to a termination/primal/dual triplet...

File: src/MOI/attributes.jl (hunk 1 above)
Comment (joaquimg) [inline src/MOI/attributes.jl]: Yep, they are the **linprog.jl** tests in [MOI#23](https://github.com/JuliaOpt/MathOptInterface.jl/issues/23#issue-242388482)

File: src/MOI/attributes.jl (hunk 2 above)
Comment (joaquimg) [inline src/MOI/attributes.jl]: not at all, I had some trouble with that. We should have a way to return extra comments about other termination/primal/dual

**Pull request merged**